applications/hpf/gpio/README.rst

```diff
@@ -1,20 +1,20 @@
-.. _sdp_gpio:
```

Comment:
I think it is best to retain this label.

Reply:
Could you explain why? High-Performance Framework (HPF) is the new official name and Software Defined Peripheral (SDP) should not be used anymore.

Reply:
It is a label and not visible in docs. If we retain it, the advantage is that we need not edit old release notes. Usually when we rename a page, the older labels are retained.

Reply:
Ok, changed back

Reply:
What I meant is we could have both the labels, retain the old and add the new one for future use. :)

Reply:
Added new label and restored the old one.